job-dsls/jobs/kie/main/compile_downstream_build_7_x.groovy

```diff
@@ -118,11 +118,7 @@ for (repoConfig in REPO_CONFIGS) {
                 triggers {
                     ghprbTrigger {
                         // execute CDB for drools, droolsjbpm-knowledge and appformer by default
-                        if ((repo != "drools") && (repo != "droolsjbpm-knowledge") && (repo != "appformer")) {
-                            onlyTriggerPhrase(true)
-                        } else {
-                            onlyTriggerPhrase(false)
-                        }
+                        onlyTriggerPhrase(false)
```

Comment:
```suggestion
                       // execute CDB for all repositories  by default unless it is a leaf repository
                      def repository_leafs = ["lienzo-tests", "kie-wb-distributions", "optaplanner-wb", "jbpm-wb", "openshift-drools-hacep", "process-migration-service", "optaweb-employee-rostering", "optaweb-vehicle-routing", "kie-docs"]
                      if (repository_leafs.contains(repo)) {
                          onlyTriggerPhrase(false)
                      } else {
                          onlyTriggerPhrase(true)
                      }
```

Reply:
i added possible logic for not including leafs repo. But this is just hand coded without checked syntax, so can be wrong ;)

Reply:
could we try this suggestion? @baldimir do you agree?